### PR TITLE
Add parameters for Snow, MedlynConductance, and BeerLambert

### PR DESCRIPTION
**Pull request by nefrathenrici**
I bundled the remaining parameters in this PR
- SnowParameters
- MedlynConductanceParameters
- BeerLambertParameters and TwoStreamParameters

I can split this PR up if it's too large
I will remove the aliases before merging.

### CODE REVIEW

**Comment by kmdeck** [comment]
@nefrathenrici is this PR still needing review or is it now split into smaller ones?

**Comment by nefrathenrici** [comment]
@kmdeck This doesn't need review, I'm splitting it into smaller PRs

File: src/parameters.toml [hunk 1]
```diff
@@ -275,6 +275,109 @@ value = 1
...
+type = "float"
+description = "Typical wavelength per PAR photon (m)"
+
+[wavelength_per_NIR_photon]
+alias = "λ_γ_NIR"
+value = 1.65e-6
+type = "float"
+description = "Typical wavelength per NIR photon (m)"
+
+[PAR_leaf_reflectance]
```
Comment (kmdeck) [inline src/parameters.toml:336]: reflectance and transmittance will be from maps (plant type dependent)

File: src/parameters.toml [hunk 2]
```diff
@@ -275,6 +275,109 @@ value = 1
...
+
+[snow_cover_fraction_crit_threshold]
+alias = "fS_c"
+value = 0.2
+type = "float"
+description = "Critical threshold for snow cover fraction (m)"
+
+# Two Stream/Beer Lambert Parameters
+
+[leaf_angle_distribution_function]
```
Comment (kmdeck) [inline src/parameters.toml:318]: I believe this is ultimately coming from maps too and is no longer a fixed constant for all vegetation.

File: src/parameters.toml [hunk 3]
```diff
@@ -182,6 +181,7 @@ type = "float"
 description = "Maximum depth of the dry soil layer that develops under evaporation; communication with Dani Or."
 
 [emissivity_bare_soil]
+# This parameter value doesn't match value in ClimaLand src, 0.98
 value = 0.96
```
Comment (kmdeck) [inline src/parameters.toml:185]: 0.96 is correct, thanks!

File: src/parameters.toml [hunk 4]
```diff
@@ -275,6 +275,79 @@ value = 1
...
+alias = "λ_γ_NIR"
+value = 1.65e-6
+type = "float"
+description = "Typical wavelength per NIR photon (m)"
+
+[canopy_emissivity]
+alias = "ϵ_canopy"
+value = 0.97
+type = "float"
+description = "Emissivity of the canopy"
```
Comment (kmdeck) [inline src/parameters.toml:334]: from CLM also

File: src/parameters.toml [hunk 5]
```diff
@@ -275,6 +275,79 @@ value = 1
...
+alias = "α_snow"
+value = 0.8
+type = "float"
+description = "Albedo of snow (unitless)"
+
+[snow_emissivity]
+alias = "ϵ_snow"
+value = 0.97
+type = "float"
+description = "Emissivity of snow (unitless)"
```
Comment (kmdeck) [inline src/parameters.toml:296]: from CLM also

**Review by kmdeck — APPROVED** [review]
I think this looks good. We will eventually parameterize the snow model albedo and density differently, as well as  the canopy emissivity, but having a globally constant option is probably not a bad idea either.

**Pull request merged**